src/axes/categoryAxis.ts

```diff
@@ -50,7 +50,7 @@ namespace Plottable.Axes {
 
     protected _setup() {
       super._setup();
-      this._measurer = new SVGTypewriter.Measurers.CacheCharacterMeasurer(this._tickLabelContainer);
+      this._measurer = new SVGTypewriter.CacheMeasurer(this._tickLabelContainer);
```

Comment:
Later on in this file we call _measurer.reset() whenever the chart is redrawn:

https://github.com/palantir/plottable/blob/bd/update-svg-typewriter/src/axes/categoryAxis.ts#L345

Will this blow away our caching improvements?